retrofit/src/test/java/retrofit/RequestBuilderTest.java

```diff
@@ -1041,6 +1043,8 @@
     assertThat(bodyString)
         .contains("name=\"kit\"")
         .contains("\r\nkat\r\n--");
+
+    assertThat(bodyString).doesNotContain("name=\"foo\"\r\n");
```

Comment:
nit: You could probably cover more regression cases by just checking for `doesNotContain("foo")` . . . that way if it regresses with say `\n` as the line ending, the test still fails.